### DISCUSSION TITLE
How to reuse landing page template for sections landing page

### DISCUSSION THREAD

**Discussion by ezzle** [colinwilson/lotusdocs#162] [Q&A]
Hi 
I need to reuse the landing page to render a local landing page for sections.
Is it possible? How to ?

Given the following content organization:  
docs/
docs/section1
docs/section1//section1_1
docs/section2
....

I need a local landing page for each section, reusing the index.html layout. How to ?
Adaptations needed? Which adaptations?

Thank you in advance.

Best regards

Tshitshi

**Comment by dannycowlishaw** [comment]
Hi, did you have any luck with this?
